src/main/asciidoc/index.adoc

```diff
@@ -141,6 +141,22 @@ as non-acknowledged (details can be found in the passed cause).
 
 The {@link io.vertx.ext.amqp.AmqpSender} can be used as a write stream. The flow control is implemented using AMQP credits.
 
+== Implementing request-reply
+
+To implement a request-reply behavior, you need dynamic receiver and anonymous sender. A dynamic receiver is not associated with
```

Comment:
This isnt necessarily true. Request-reply can also be done with fixed addresses and there are cases that is desirable and/or required.